capstone/src/main/java/com/google/sps/Objects/Post.java

```diff
@@ -4,18 +4,18 @@
 
 public final class Post {
 
-  private final User author;
+  private final String authorId;
```

Comment:
Question, why did we switch over to authorId instead of User? I thought during the standup on Thursday we had decided to user the Objects we were creating which will have the necessary ID and won't require you to get the data from the database because you'll already have it.

Reply:
Above Himani mentioned that we don't need all of the user's data to be loaded with each post, only the authorId and their name. I was planning on using the authorId to get their name when a post is submitted. Also open to changing it back to a User object.

Reply:
Yeah sorry that's my bad, I made that comment before our discussion yesterday! Within the database we would just store the user's id but in the actual object it makes sense to just have it be a User object.

Reply:
Yeah if you could change it back that would be great!